pkg/controller/admissionchecks/multikueue/externalframeworks/adapter.go

```diff
@@ -160,13 +160,6 @@ func (a *Adapter) DeleteRemoteObject(ctx context.Context, remoteClient client.Cl
 	return client.IgnoreNotFound(remoteClient.Delete(ctx, obj, client.PropagationPolicy(metav1.DeletePropagationBackground)))
 }
 
-// KeepAdmissionCheckPending returns false,
-// indicating that admission checks should not be kept pending by default.
-// This can be overridden by specific adapters if needed.
-func (a *Adapter) KeepAdmissionCheckPending() bool {
```

Comment:
Yeah, I like getting rid of this function, but please don't couple to this bugfix PR, as mentioned in https://github.com/kubernetes-sigs/kueue/pull/8189/changes#r2622191492